doc/README.md

```diff
@@ -0,0 +1,2 @@
+Information: This directory is not a documentation source any more. Developer docs are contained in 
```

Comment:
I had this change already on my TODO list. I opened a pull request for removing this obsolete directory.